config/webpacker.yml

```diff
@@ -36,6 +36,7 @@ development:
 
 test:
   <<: *default
+  compile: true
```

Comment:
This is apparently necessary to pass the 2.5 circleci config, since I touched `simple_form.en.yml`